src/parsers/css/cssom.js

```diff
@@ -10,6 +10,7 @@ export default {
   displayName: ID,
   version: pkg.version,
   homepage: pkg.homepage,
+  locationProps: ['__starts', '__ends'],

```

Comment:
How about using a `Set` here instead? I know it will probably introduce more overhead but it would be more in line with `_ignoredProperties`.